introduction.tex

```diff
@@ -50,6 +50,19 @@ \section{Introduction}
 
 \noindent
 \textbf{Our contributions.}
+A summary of our contributions is as follows:
+\begin{enumerate}[wide, labelwidth=!, labelindent=0pt, label=(\roman*)]
+    \item \textbf{Primitive definition.} Our definitional contribution introduces Proof-of-Burn as a cryptographic primitive for the first time. We
+    define it as a protocol which consists of two algorithms, a burn address \emph{generator} and a burn address \emph{verifier}. We put forth the foundational properties which make for secure burn protocols, namely \emph{correctness}, \emph{unspendability}, \emph{binding}, and \emph{uncensorability}.
+    \item \textbf{Novel construction.} We propose a novel and simple construction which is flexible and can be adapted to be used in any of the
+    existing cryptocurrencies, as long as they use public key hashes for address generation. To our knowledge, all popular cryptocurrencies are
+    compatible with our scheme. We prove our construction secure in the Random Oracle model.
+    \item \textbf{Experimental results.} We provide a compehensively tested production-grade implementation of our burn verifier in Ethereum
```

Comment:
Maybe let's add a word about cross-chain consumption at the end of this paragraph:

> This allows coins burned on one blockchain to be consumed on another for the purposes of, for example, ERC-20 tokens creation.

with a citation to ERC-20 (see PoW sidechains for appropriate \cite)